erdpy_network/transaction_completion_strategy.py

```diff
@@ -16,6 +27,36 @@ def is_completed(self, transaction: ITransactionOnNetwork) -> bool:
         return not transaction.get_status().is_pending()
 
 
+# this class is similar to the one in erdjs-network-providers
 class TransactionCompletionStrategyOnProxy:
-    def is_completed(self, transaction: ITransactionOnNetwork):
-        raise NotImplementedError('Method is not implemented yet')
+    def is_completed(self, transaction: ITransactionOnNetwork) -> bool:
+        if transaction.get_status().is_pending():
```

Comment:
I do not see `get_status()` on the internal interface `ITransactionOnNetwork`.

Reply:
It's here:
https://github.com/ElrondNetwork/sdk-erdpy-eggs-network-providers/blob/get-transaction-proxy/erdpy_network/transaction_completion_strategy.py#L21